src/lib/operators/print.cpp

```diff
@@ -83,6 +86,15 @@ std::shared_ptr<const Table> Print::_on_execute() {
       continue;
     }
 
+    // print the encoding information
+    for (ColumnID column_id{0}; column_id < chunk->column_count(); ++column_id) {
+      const auto column_width = widths[column_id];
+      const auto& segment = chunk->get_segment(column_id);
+      _out << "|" << std::setw(column_width - 2) << _segment_type(segment) << std::setw(0);
```

Comment:
Why `column_width - 2`?

Reply:
Good catch. That was from a time where `<>` were already included.